shared/metrics/src/costs/cost-calculator.ts

```diff
@@ -27,7 +27,7 @@ export class CostCalculator {
 
         return acc;
       }, new CostReport()),
-      last(null, null, new CostReport())
+      last(null, new CostReport())
```

Comment:
This is an API change in RxJS 6.x

With result selector in 5.x

```
source.pipe(
 last(predicate, resultSelector, defaultValue)
) 
```

Without in 6.x

```
source.pipe(
 last(predicate, defaultValue),
 map(resultSelector)
)
```

Reply:
Thanks for explaining these changes @PascalPrecht! :tada: